### PR TITLE
Interact with 3D meshes

### PR DESCRIPTION
**Pull request by aChanEP**
Description:

1.) Mesh does not show up when in fullscreen mode.  For some reason it disappears DONE
2.) Work on getting cursor to show up only for devices without controls DONE

This PR is part of these three tasks:

https://elasticpath.atlassian.net/secure/RapidBoard.jspa?rapidView=467&projectKey=RS&modal=detail&selectedIssue=RS-701

https://elasticpath.atlassian.net/secure/RapidBoard.jspa?rapidView=467&projectKey=RS&modal=detail&selectedIssue=RS-702

https://elasticpath.atlassian.net/secure/RapidBoard.jspa?rapidView=467&projectKey=RS&modal=detail&selectedIssue=RS-685

Also adjusted the icons for VR and AR.

Linting:

- [x] No linting errors

Component Updates:

- [x] Component package requires update on npm [@elasticpath/store-components](https://www.npmjs.com/package/@elasticpath/store-components)

Tests:

- [ ] E2E tests (npm test run with `e2e`)
- [ ] Manual tests

Documentation:

- [ ] Requires documentation updates

### CODE REVIEW

File: src/components/src/ProductDisplayItem/productdisplayitem.main.tsx [hunk 1]
```diff
@@ -276,10 +282,12 @@ class ProductDisplayItemMain extends Component<ProductDisplayItemMainProps, Prod
...
       const itemLookupRes = await itemLookup(productId, false);
       let arFileExists = false;
-      let vrFileExists = false;
+      let backgroundVRImageExists = false;
+      let meshVRImageExists = false;
 
       if (Config.vr.enable) {
-        vrFileExists = await ProductDisplayItemMain.urlExists(Config.vr.skuVrImagesUrl.replace('%sku%', itemLookupRes._code[0].code));
+        backgroundVRImageExists = await ProductDisplayItemMain.urlExists(Config.vr.skuVrImagesUrl.replace('%sku%', itemLookupRes._code[0].code));
+        meshVRImageExists = await ProductDisplayItemMain.urlExists(Config.vr.skuVrMeshesUrl.replace('%sku%', itemLookupRes._code[0].code));
```
Comment (shaunmaharaj) [inline src/components/src/ProductDisplayItem/productdisplayitem.main.tsx]: excellent 👍

File: src/components/src/VRProductDisplayItem/README.md [hunk 2]
```diff
@@ -12,6 +12,12 @@ For an example of how to integrate this component into your own project checkout
 
 `productdisplayitem.main.tsx`
 
+The component gives the user the ability to rotate in the vr-scene with mobile, dedicated vr devices and on desktop.  Look at the aframe website for a more exhaustive list.
```
Comment (shaunmaharaj) [inline src/components/src/VRProductDisplayItem/README.md]: We should provide a link to the A-Frame website here if we're referencing it.

Comment (shaunmaharaj) [inline src/components/src/VRProductDisplayItem/README.md]:
```suggestion
The component gives the user the ability to rotate in the vr-scene with mobile devices, dedicated vr devices, and on desktop.  Look at the aframe website for a more exhaustive list.  
```

File: src/components/src/VRProductDisplayItem/README.md [hunk 3]
```diff
@@ -12,6 +12,12 @@ For an example of how to integrate this component into your own project checkout
 
 `productdisplayitem.main.tsx`
 
+The component gives the user the ability to rotate in the vr-scene with mobile, dedicated vr devices and on desktop.  Look at the aframe website for a more exhaustive list.  
+The component also allows the user to set a panorama image, a 3D mesh, and an html markup that populates a 3D plane.  
+
+Note that the html markup that populates the plane in the 3d scene
+is hidden outside of aframes `Scene` tag.  Populate markup inside the div with the classname `inner-html-texture` to customize desired information to be shown in the 3D scene.
```
Comment (shaunmaharaj) [inline src/components/src/VRProductDisplayItem/README.md]: Unnecessary new line.

Comment (shaunmaharaj) [inline src/components/src/VRProductDisplayItem/README.md]: Very well described!

File: src/components/src/VRProductDisplayItem/README.md [hunk 4]
```diff
@@ -12,6 +12,12 @@ For an example of how to integrate this component into your own project checkout
...
+
```
Comment (shaunmaharaj) [inline src/components/src/VRProductDisplayItem/README.md]: Remember to mention the meshURL in the example snippet.

File: src/components/src/VRProductDisplayItem/VRProductDisplayItem.less [hunk 5]
```diff
@@ -143,3 +146,40 @@
...
+    height: 100%; 
+    position: fixed; 
+    left: 0; 
+    top: 0; 
+    overflow: hidden;
+    z-index: -2;
+}
+
+.inner-html-texture {
+    // background-image: url(cheesecake.png);
```
Comment (shaunmaharaj) [inline src/components/src/VRProductDisplayItem/VRProductDisplayItem.less]: leftover cheesecake 🍰

File: src/components/src/VRProductDisplayItem/VRProductDisplayItem.less [hunk 6]
```diff
@@ -143,3 +146,40 @@
...
+    color: white; 
+    width: 500px; 
+    height: 500px; 
+    font-size: 64px; 
+    font-family: monospace; 
+    text-align: center
+}
+
+.html-text-texture {
+    background: rgb(30, 30, 30);
```
Comment (shaunmaharaj) [inline src/components/src/VRProductDisplayItem/VRProductDisplayItem.less]: Should this color be them-able?

File: src/components/src/VRProductDisplayItem/VRProductDisplayItem.tsx [hunk 7]
```diff
@@ -66,24 +190,26 @@ class VRProductDisplayItem extends Component<IVRComponentProps, IVRComponentStat
...
+          <div className="cover-html-texture" />
+
+          {/** The place holder html for description panel.  Replace this div to adjust what is shown! */}
+          <div className="outer-html-texture">
+            <div id="boxHTML" className="inner-html-texture">
+              <p className="html-text-texture">Place holder html</p>
+            </div>
+          </div>
+
+          <Scene className="vr-container" embedded vr-mode-ui="enterVRButton: #myEnterVRButton;" background="color: grey" loading-screen="backgroundColor: #000000; dotsColor: white">
```
Comment (shaunmaharaj) [inline src/components/src/VRProductDisplayItem/VRProductDisplayItem.tsx]: There's a lot of inline css here. Can we move it to the less file?

File: src/ep.config.json [hunk 8]
```diff
@@ -1,7 +1,7 @@
 {
   "cortexApi": {
     "path": "/cortex",
-    "scope": "vestri",
+    "scope": "vestri_reference",
```
Comment (shaunmaharaj) [inline src/ep.config.json]: Reminder to revert this 😄

File: src/ep.config.json [hunk 9]
```diff
@@ -58,7 +58,8 @@
   },
   "vr": {
     "enable": true,
-    "skuVrImagesUrl": "https://s3.amazonaws.com/referenceexp/vr/%sku%.jpg"
+    "skuVrImagesUrl": "https://s3.amazonaws.com/referenceexp/vr/%sku%.jpg",
```
Comment (shaunmaharaj) [inline src/ep.config.json]: I'll add some documentation for this tomorrow 👍

File: src/components/src/VRProductDisplayItem/README.md (hunk 2 above)
Comment (aChanEP) [reply src/components/src/VRProductDisplayItem/README.md]: Sounds good

File: src/components/src/VRProductDisplayItem/VRProductDisplayItem.less (hunk 6 above)
Comment (aChanEP) [reply src/components/src/VRProductDisplayItem/VRProductDisplayItem.less]: Yeah it might be better to make it a seperate react component maybe?

File: src/components/src/VRProductDisplayItem/VRProductDisplayItem.tsx (hunk 7 above)
Comment (aChanEP) [reply src/components/src/VRProductDisplayItem/VRProductDisplayItem.tsx]: Not so sure styles will be able to attach attributes to an element.  Those are aframe specific attributes that dictate settings in the scene.

Comment (aChanEP) [reply src/components/src/VRProductDisplayItem/VRProductDisplayItem.tsx]: Best I can do is maybe format it a little better?

File: src/components/src/VRProductDisplayItem/README.md (hunk 4 above)
Comment (aChanEP) [reply src/components/src/VRProductDisplayItem/README.md]: done

File: src/components/src/VRProductDisplayItem/VRProductDisplayItem.less (hunk 5 above)
Comment (aChanEP) [reply src/components/src/VRProductDisplayItem/VRProductDisplayItem.less]: removed

File: src/components/src/VRProductDisplayItem/VRProductDisplayItem.less (hunk 6 above)
Comment (aChanEP) [reply src/components/src/VRProductDisplayItem/VRProductDisplayItem.less]: Made it into its own react component so someone can come in and more easily change it

File: src/components/src/VRProductDisplayItem/VRProductDisplayItem.tsx (hunk 7 above)
Comment (aChanEP) [reply src/components/src/VRProductDisplayItem/VRProductDisplayItem.tsx]: I just reformatted it so its easier to read

File: src/components/src/VRProductDisplayItem/README.md (hunk 2 above)
Comment (aChanEP) [reply src/components/src/VRProductDisplayItem/README.md]: There is a reference above in the README.  Ill reword it to reference it more clearly

File: src/components/src/VRProductDisplayItem/README.md (hunk 3 above)
Comment (aChanEP) [reply src/components/src/VRProductDisplayItem/README.md]: I've reworded it.  The newline seems to make sense to me.  Let me know if it still doesn't

File: src/ep.config.json (hunk 8 above)
Comment (aChanEP) [reply src/ep.config.json]: reverted

**Comment by aChanEP** [comment]
> Wonderful job with this one Alvin! Thanks for completing this 😃
> 
> Added a few minor comments. We can merge once it's set.

addressed all comments! let me know if u have any other concerns!
